Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.yml

```diff
@@ -177,7 +177,7 @@ configuration:
   advanced: true
   required: false
 - display: Operations to fetch
-  additionalinfo: A comma-separated list of the operations you want to fetch. Content records with an operation that is not specified will not be fetched. If this field is left empty, all operations will be fetched.
+  additionalinfo: Important, this field is case-sensitive. A comma-separated list of the operations you want to fetch. Content records with an operation that is not specified will not be fetched. If this field is left empty, all operations will be fetched.
```

Comment:
Is the list-activities command helpful here as well? 
If so, let's add that the user can run the command first to see what activities/operations are available to them.
If so let's add to the readme as well